### PR TITLE
update turboJPEG installation

### PR DESCRIPTION
**Pull request by LakshmiKumar23**

### CODE REVIEW

**Comment by kiritigowda** [comment]
@LakshmiKumar23 it would be good to add RPP deps before we install RPP. This way we won't have any missing links. Can you add the line below in this commit before the RPP install line.

```
 sudo apt --yes install libboost-all-dev clang
```

**Comment by LakshmiKumar23** [comment]
@kiritigowda updated the script

**Comment by kiritigowda** [comment]
@LakshmiKumar23 did you test this changes?

**Comment by LakshmiKumar23** [comment]
@kiritigowda yes.

**Comment by kiritigowda** [comment]
Can we add JSON to this commit?

**Comment by kiritigowda** [comment]
Let us run this script in the rocm-docker and build MIVisionX. Then merge this in.

**Comment by LakshmiKumar23** [comment]
@kiritigowda I am running it right now

File: MIVisionX-setup.py [hunk 1]
```diff
@@ -185,6 +185,8 @@
 	os.system('sudo '+linuxFlag+' '+linuxSystemInstall+' -y '+linuxSystemInstall_check+' install qt5-default qtcreator')
 	# Install RPP
 	if rppInstall == 'yes':
+		os.system('(sudo apt-get install libjsoncpp-dev)')
+		os.system('(sudo apt --yes install libboost-all-dev clang)')
```
Comment (kiritigowda) [inline MIVisionX-setup.py:189]: apt/apt-get does not work on CentOS, to use the install pattern in line 185

Comment (kiritigowda) [reply MIVisionX-setup.py:189]: @LakshmiKumar23 any updates on this PR?

Comment (LakshmiKumar23) [reply MIVisionX-setup.py]: Updated for cent-os. Tested on both platforms.

File: MIVisionX-setup.py [hunk 2]
```diff
@@ -185,13 +185,17 @@
...
 		#Yasm/Nasm for TurboJPEG
 		if linuxSystemInstall == 'apt-get':
 			os.system('sudo -v')
 			os.system('sudo '+linuxFlag+' '+linuxSystemInstall+' -y '+linuxSystemInstall_check+' install nasm yasm')
+			#json-cpp
+			os.system('sudo -v')
+			os.system('sudo '+linuxFlag+' '+linuxSystemInstall+' -y '+linuxSystemInstall_check+' install libjsoncpp-dev')
+			#clang+boost
+			os.system('sudo -v')
+			os.system('sudo '+linuxFlag+' '+linuxSystemInstall+' -y '+linuxSystemInstall_check+' install libboost-all-dev clang')
```
Comment (kiritigowda) [inline MIVisionX-setup.py:198]: boost is required by RPP, needs to be above RPP install

Comment (kiritigowda) [reply MIVisionX-setup.py:198]: @LakshmiKumar23 boost and clang for RPP

**Comment by kiritigowda** [comment]
@LakshmiKumar23 looks good. Let me know when the testing is done. I will merge this.

**Pull request merged**